runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -122,12 +122,14 @@ WikiConfig getConfig(ConfigManager configManager)
 	@Override
 	public void startUp()
 	{
+		spriteManager.addSpriteOverrides(WikiSprite.values());
```

Comment:
This should only be added when the widget is added

Reply:
I'm sorry, I guess I don't quite understand. This is the same as the original code, and is only done on startup so I thought it's happening exclusively when it's added?

Reply:
this change is entirely about not adding the widget sometimes, so the sprites should only be added under the same condition, as such the sprites should be removed when the widget is removed

Reply:
You added a toggle to not show the widget, and in that case you would not need the sprite overrides. What it should do is add the overrides whenever the widget is toggled to show, and remove whenever it gets hidden.

Reply:
Wait you're commenting on an outdated change